src/app/util/af-types.h

```diff
@@ -116,7 +117,9 @@ typedef struct
      * Total number of events supported by the cluster instance (in eventList array).
      */
     uint16_t eventCount;
-} EmberAfCluster;
+
+    bool IsServerMask() const { return (mask & CLUSTER_MASK_SERVER) != 0; }
```

Comment:
```suggestion
    bool IsServerCluster() const { return (mask & CLUSTER_MASK_SERVER) != 0; }
```
or maybe `IsServer()`.  The fact that there is a mask is an implementation detail.

Reply:
Used IsServer since this is on `EmberAfCluster` so cluster seems implied.